### PR TITLE
add transport mode and vehicle, models and views

### PR DESCRIPTION
**Pull request by lepistone**
- [x] models
- [x] views
- [x] doc
- [x] i18n

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

File: transport_information/README.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Transport Information
+=====================
+This simple module is en extraction of objects related to transportation of
```
Comment (gurneyalex) [inline transport_information/README.rst]: s/en/an/

File: transport_information/model/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+import transport_mode
+import transport_vehicle
```
Comment (gurneyalex) [inline transport_information/model/__init__.py]: please use relative imports

File: transport_information/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_transport_mode,access_transport_mode,model_transport_mode,,1,1,1,1
+access_transport_vehicle,access_transport_vehicle,model_transport_vehicle,,1,1,1,1
```
Comment (gurneyalex) [inline transport_information/security/ir.model.access.csv]: I'm concerned that such permissions could allow e.g. portal users to create transport modes and vehicles. IMO, by default only purchase managers should be allowed to create such records (and anyone can read them)

Comment (foutoucour) [inline transport_information/security/ir.model.access.csv]: I agree here.
I would add to the suggestions from @gurneyalex that only admin should be able to unlink.

**Comment by lepistone** [comment]
I addressed the points raised. I gave purchase managers (not admin) the right do delete those.

Thanks for the review.

**Pull request merged**